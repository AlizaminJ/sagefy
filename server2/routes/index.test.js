const request = require('supertest')
const express = require('express')
const indexRoutes = require('./index')

describe('indexRoutes', () => {
  const app = indexRoutes(express())

  describe('GET /x', () => {
    test('Should send a message', () =>
      request(app)
        .get('/x')
        .expect(200)
        .expect('Content-Type', /json/)
        .expect(res => expect(res.text).toMatchSnapshot()))
  })

  describe('GET /x/sitemap', () => {
    test('should create a sitemap', () => {
      expect(true).toBe(false)
    })
  })

  describe('GET /x/search', () => {
    test('should get search results', () => {
      expect(true).toBe(false)
    })
  })
})
