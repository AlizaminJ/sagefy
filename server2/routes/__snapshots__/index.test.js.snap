// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`indexRouter GET /x Should send a message 1`] = `"{\\"message\\":\\"Welcome to the Sagefy service!\\"}"`;
